src/bp-core/deprecated/1.1.php

```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+ * Deprecated Functions
+ *
+ * @package BuddyPress
+ * @deprecated 1.1.0
```

Comment:
You need to rename this file in favor of 15.0.php.

```suggestion
 * @deprecated 15.0.0
```